handlers/isClientExist.js

```diff
@@ -0,0 +1,12 @@
+const isClientExist = async (Model, req) => {
+    // check for existing employee
+    const { company, managerSurname, managerName, email, phone } = req.body
+    let isExist;
+    if (company || managerSurname || managerName || email || phone) {
+        isExist = await Model.findOne({ company, managerName, managerName, email, phone })
```

Comment:
I see you mention `managerName` twice. please change this 🙏

Reply:
My bad. Let me change it.